### ISSUE TITLE
Misaligned tooltips on v3.4.1

### ISSUE THREAD

**Issue by azdanov** [johnlindquist/AceJump#197]
3.4.1: <img width="1392" alt="screen shot 2017-10-15 at 18 26 36" src="https://user-images.githubusercontent.com/6123841/31586319-e83dc204-b1d7-11e7-804a-207baa35b9e3.png">
3.4.0: 
<img width="1280" alt="screen shot 2017-10-15 at 18 35 37" src="https://user-images.githubusercontent.com/6123841/31586330-fb4ac14e-b1d7-11e7-8611-ca9668cc2e3d.png">

**Comment by breandan** [comment]
Which version of WebStorm are you running?

**Comment by azdanov** [comment]
WebStorm 2017.2.4
Build #WS-172.4155.35, built on September 11, 2017

JRE: 1.8.0_152-release-915-b11 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13

**Comment by breandan** [comment]
Will investigate, thanks. I am running:

IntelliJ IDEA 2017.3 EAP (Ultimate Edition)
Build #IU-173.3188.16, built on October 10, 2017
IntelliJ IDEA EAP User
Expiration date: November 9, 2017
JRE: 1.8.0_152-release-1024-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.12.6

![screen shot 2017-10-15 at 12 13 48 pm](https://user-images.githubusercontent.com/175716/31586704-785388d2-b1a2-11e7-9894-3582f5480617.png)

**Comment by azdanov** [comment]
Same with IntelliJ, v3.4.0 is working fine.

IntelliJ IDEA 2017.2.5
Build #IU-172.4343.14, built on September 26, 2017

JRE: 1.8.0_152-release-915-b12 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13

<img width="1280" alt="screen shot 2017-10-15 at 19 18 20" src="https://user-images.githubusercontent.com/6123841/31586734-c3983fc8-b1dd-11e7-995c-c65ac5de4f17.png">

**Comment by breandan** [comment]
I think there was a platform change between `2017.2.5` and `2017.3 EAP` that changes how things are rendered. Can you reproduce on `2017.3 EAP`?

**Comment by azdanov** [comment]
No problem on EAP.

IntelliJ IDEA 2017.3 EAP (Ultimate Edition)
Build #IU-173.3188.16, built on October 10, 2017
IntelliJ IDEA EAP User
Expiration date: November 9, 2017
JRE: 1.8.0_152-release-1024-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13

<img width="1280" alt="screen shot 2017-10-15 at 19 28 36" src="https://user-images.githubusercontent.com/6123841/31586806-35f3e850-b1df-11e7-8467-7c69baa28757.png">

**Comment by breandan** [comment]
Weird. Okay, so this is definitely related to a change between [`idea/172.4343.14`](https://github.com/JetBrains/intellij-community/releases/tag/idea%2F172.4343.14) and [`idea/173.3188`](https://github.com/JetBrains/intellij-community/releases/tag/idea%2F173.3188)

**Comment by breandan** [comment]
Prior to AceJump 3.4.1 we used to add the `Canvas` like so:

```
contentComponent.add(Canvas)
Canvas.setBounds(0, 0, contentComponent.width, contentComponent.height)
val loc = convertPoint(Canvas, location, component.rootPane)
Canvas.setLocation(-loc.x, -loc.y)
```

Now, to account for IntelliJ 2017.3 EAP we are just doing:

```
contentComponent.add(Canvas)
Canvas.setBounds(0, 0, contentComponent.width, contentComponent.height)
```

For some reason, prior to `2017.3 EAP` we needed to invert the coordinates. But why?

**Comment by breandan** [comment]
This issue should be fixed in `3.4.2`. Still not sure what's causing it, but thanks for reporting.

**Comment by azdanov** [comment]
No problems with [`3.4.2`](https://plugins.jetbrains.com/plugin/7086-acejump/update/39594).
